tests/compile/test_builders.py

```diff
@@ -584,6 +585,22 @@ def test_explicit_input_from_shared(self):
         out = test_ofg(y, y)
         assert out.eval() == 4
 
+    def test_repeated_inputs(self):
+        x = pt.dscalar("x")
+        y = pt.dscalar("y")
+
+        with pytest.raises(
+            ValueError,
+            match="There following variables were provided more than once as inputs to the "
+            "OpFromGraph",
+        ):
+            OpFromGraph([x, x], [x + y])
```

Comment:
Why not make it comparable to before, to highlight the only change is the double input? This also has a missing input
```suggestion
            OpFromGraph([x, x, y], [x + y])
```

Reply:
ah good catch. I'm going too fast and being sloppy.